plugin/src/main/java/io/fabric8/maven/plugin/mojo/infra/AbstractInstallMojo.java

```diff
@@ -282,16 +309,16 @@ protected Platform getPlatform() {
         }
     }
 
-    protected void runGofabric8(File command, String ... args) throws MojoExecutionException {
+    protected void executeCommand(File command, String binName, String ... args) throws MojoExecutionException {
         // Be sure to run in batch mode
         List<String> argList = new ArrayList<>(Arrays.asList(args));
-        argList.add("--batch");
+        //argList.add("--batch");
```

Comment:
From the `gofabric8` doc, the `--batch` flag was used to avoid interactive user input. It seems you've removed it from the default list of args, but you didn't put it back in every call to `executeCommand` for the GOFABRIC8 binary (e.g. see `ClusterStartMojo` and others).